### ISSUE TITLE
Blocked by CORS policy

### ISSUE THREAD

**Issue by mirrorleap** [primodotso/primo-server#28]
Hi, 

I deployed Primo Server on Netlify and everything's working smoothly. I also installed the Primo Desktop app to manage my server locally. However, adding the Primo Server is being blocked by the CORS policy. I noticed the following error in the Developer tools console:
> Access to XMLHttpRequest at 'https://xxxxx.netlify.app/api/sites' from origin 'app://-' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

I added the _headers file with the following content to the public and src directory of the repo and re-deployed, but it doesn't fix the problem:
`/*
  Access-Control-Allow-Origin: *`

Any ideas how I can resolve this issue? Thanks.

**Comment by mateomorris** [comment]
Did you end up deploying Primo Server to Netlify instead? Not sure since it seems to be working for me, but I'm actually taking down Primo Desktop for now to focus on more features for server (realtime collab, on-page editing, framework integrations) - having both kind of doubles the work and it seems like just about everyone is using server anyway - so if the issue lies there I won't be able to push a fix for it.

**Comment by mirrorleap** [comment]
Hi @mateomorris, yes I ended up deploying Primo Server on Netlify rather than Cloudflare Pages (free tier workers limitation). The build config is pretty much the same as Vercel.

Guess it does make more sense to focus on adding features to Primo Server rather than Primo Desktop. Looking at the **'CORS policy'** error, I was thinking the issue might be on the server-side. But, not sure, as sometimes CORS errors are due to client-side requests.

I suppose you are running this on Vercel? Perhaps the issue is with Netlify, in which case I will try to find a possible solution and update it here.

**Comment by mateomorris** [comment]
Yeah everyone I know runs it on vercel. Cors issues are the bane of my existence so I can't remember if this is what fixed it for vercel, but there's a [vercel.json](https://github.com/primodotso/primo-server/blob/master/vite.config.js) file in the root directory that seems to handle it. Seems like you could do the same thing by adding a [Netlify TOML](https://answers.netlify.com/t/access-control-allow-origin-policy/1813/2)? Feel free to submit a PR if that works for you.
